host/onebox.go

```diff
@@ -1042,6 +1042,9 @@ func (c *cadenceImpl) newRPCFactory(serviceName string, host membership.HostInfo
 		c.logger.Fatal("failed to get frontend PortGRPC", tag.Value(c.FrontendHost()), tag.Error(err))
 	}
 
+	directOutboundPCF := rpc.NewDirectPeerChooserFactory(serviceName, c.logger)
+	directConnRetainFn := func(opts ...dynamicconfig.FilterOption) bool { return false }
```

Comment:
I might vote for `neverRetainConns` or something else implies the behavior being selected, rather than basically the type name.

Reply:
It returns false for now. I'll change the default value to true when new behavior is implemented. Tightly coupling the closure variable name with the return value would require remembering to change the name and its occurrences later on. That sounds like minor code smell to me.